Kiosk/Help/HelpViewController.swift

```diff
@@ -120,14 +127,16 @@ private extension HelpViewController {
         stackView.addSubview(conditionsButton, withTopMargin: "\(headerMargin)", sideMargin: "\(sideMargin)")
         stackView.addSubview(privacyButton, withTopMargin: "\(inbetweenMargin)", sideMargin: "\(self.sideMargin)")
         
-        hasBuyersPremiumSignal.subscribeNext { [weak self] in
-            let hasBuyersPremium = $0 as! Bool
-            if hasBuyersPremium {
-                self?.stackView.addSubview(self!.buyersPremiumButton, withTopMargin: "\(self!.inbetweenMargin)", sideMargin: "\(self!.sideMargin)")
-            } else {
-                self?.stackView.removeSubview(self!.buyersPremiumButton)
+        hasBuyersPremiumSignal
+            .subscribeNext { [weak self] in
+                let hasBuyersPremium = $0
```

Comment:
( nitpick, not blocker ) it feels weird seeing `$0` on multi line blocks, am I alone here?